src/CoreWlan/Enums.cs

```diff
@@ -116,4 +116,20 @@ public enum CWKeychainDomain : nuint_compat_int {
 		User = 1,
 		System = 2,
 	}
+
+	[Native]
+	public enum CWEventType : nuint_compat_int {
```

Comment:
`nuint_compat_int` should not be used in new bindings. That was required only to support different signature for classic/unified (i.e. to keep compatibility between them). Newer bindings can always define the right type.

E.g.  in this case it should be `nint` as this is not a `[Flag]` decorated enum. It also match seems to match Apple's headers.